### DISCUSSION TITLE
[361280]Turmoil [仅本体]

### DISCUSSION THREAD

**Discussion by Wuli-Tang** [pjy612/SteamManifestCache#421] [Share]
`appid:361280`
keys:
```
228990;44d8c45ce229a11c4f231a3d2a350eaf80b0d69a8af938ec7ccca720f694b0e8
361281;0f3e5d3968ab0559516621bee8349b5f5a8f651f091fa419d8aadee78cfeb67c
361282;f2ff46e2f1b65bcab5960ce224a5fee831f290cee820939a636a13da0cc1fd8b
```
[361280_config.vdf.zip](https://github.com/pjy612/SteamManifestCache/files/14047959/361280_config.vdf.zip)
Code:CB2A2B57B8C8A797
Email:wuli_hui@outlook.com

**Comment by ZEERDEER** [comment]
* 清单补齐

**Reply by Wuli-Tang** [reply]
vdf里面找不到mac和Linux的密钥

**Reply by pjy612** [reply]
* 请附带对应清单

**Reply by Wuli-Tang** [reply]
第一次发，不知道对应的清单能在哪里找到

**Reply by pjy612** [reply]
[Manifest](https://github.com/pjy612/SteamManifestCache/wiki/Manifest)

**Reply by Wuli-Tang** [reply]
哦哦哦好的

**Reply by Wuli-Tang** [reply]
更新了，不知道对不对
